packages/types/src/index.ts

```diff
@@ -0,0 +1,37 @@
+export { DatabaseInterface as DBInterface } from './types/database-interface'
+import { ConnectionInformations as ConnectionInformationsType } from './types/connection-informations'
+
+// ? Tokens
+export { Tokens as TokensType } from './types/tokens'
+export { TokenRecord } from './types/token-record'
+export { Session as SessionType} from './types/session'
+
+// ? User
+export { User as UserObjectType } from './types/user'
+export { CreateUser as CreateUserType } from './types/create-user'
+
+
+export { EmailRecord } from './types/email-record'
+
+// ? Operation Results
+export { LoginResult as LoginReturnType } from './types/login-result'
+export { ImpersonationResult as ImpersonateReturnType } from './types/impersonation-result'
+
+
+// ? Operation Parameters
```

Comment:
we will need to figure this before merging no?

Reply:
Yes actually I want to discuss it ASAP

Reply:
So this will be used on the password server and password client right?

Reply:
the @accounts/types package will be used on pretty much every @accounts package, as a devDependency

Reply:
I mean the type

Reply:
Yes it is

Reply:
What is the idea of Operation Parameters?

Reply:
For several operations, the function expect one object as parameters which contains all the operation needs to be complete